mix_helpers.exs

```diff
@@ -410,11 +410,8 @@ defmodule Mix.Appsignal.Helper do
   # If `ldd` is not found it returns `nil`
   defp ldd_version_output do
     case @system.cmd("ldd", ["--version"], stderr_to_stdout: true) do
```

Comment:
Do we still needs this pattern matching? Or can we directly return the output (And rescue if something went wrong)

Reply:
Yes, because we return a different tuple than `System.cmd`, `:ok` and `:error` instead of exit codes. See also the `rescue` block.
We shouldn't have to rely on the exit codes further up the chain.